package.json

```diff
@@ -105,10 +105,11 @@
     "@angular/platform-browser-dynamic": "^8.0.0",
     "@angular/router": "^8.0.0",
     "@angular/upgrade": "^8.0.0",
+    "@ng-bootstrap/ng-bootstrap": "^5.3.0",
     "angular": "^1.7.9",
     "angular-cookies": "^1.7.9",
     "angular-route": "^1.7.9",
-    "bootstrap": "^3.4.1",
+    "bootstrap": "^4.0.0",
```

Comment:
Doesn't this break the styles?

Reply:
It would but we already have bootstrap4 in manifest.json

Reply:
Oh, I see.